src/components/editor/MonacoEditor.vue

```diff
@@ -60,7 +61,12 @@ async function updateFile() {
   };
 
   await writeProjectFile(props.projectName, file);
-  FileEvent.UpdateEditorContentEvent.next(file.id);
+  const [fileStatus] = await getStatus(
+    props.projectName,
+    [file.id],
+    (path) => path === file.id,
+  );
+  FileEvent.UpdateEditorContentEvent.next(fileStatus);
```

Comment:
Update doc for `UpdateEditorContentEvent` inside `FileEvent.js`

Reply:
Done

Reply:
you updated `UpdateFileContentEvent` instead of `UpdateFileContentEvent`

Reply:
done